scripts/bootloader/provision.py

```diff
@@ -38,6 +40,7 @@ def parse_args():
                         help="Semicolon-separated list of public key .pem files.")
     parser.add_argument("-o", "--output", required=False, default="provision.hex",
                         help="Output file name.")
+    parser.add_argument("--num-counter-slots", required=True, help="Number of monotonic counter slots.")
```

Comment:
Could we have this default=0, and then not required?

Reply:
Will do